addons/mail/static/src/components/message/message.xml

```diff
@@ -29,7 +29,7 @@
                     <div class="o_Message_sidebar" t-att-class="{ 'o-message-squashed align-items-start': messageView.isSquashed }">
                         <t t-if="!messageView.isSquashed">
                             <div class="o_Message_authorAvatarContainer o_Message_sidebarItem">
-                                <img class="o_Message_authorAvatar rounded-circle" t-att-class="{ o_Message_authorRedirect: messageView.hasAuthorOpenChat, o_redirect: messageView.hasAuthorOpenChat }" t-att-src="messageView.message.avatarUrl" t-on-click="_onClickAuthorAvatar" t-att-title="messageView.hasAuthorOpenChat ? messageView.authorAvatarTitleText : ''" alt="Avatar"/>
+                                <img class="o_Message_authorAvatar rounded-circle" t-att-class="{ o_Message_authorRedirect: messageView.hasAuthorOpenChat, o_redirect: messageView.hasAuthorOpenChat }" t-att-src="messageView.message.avatarUrl" role="button" tabindex="0" t-on-click="_onClickAuthorAvatar" t-att-title="messageView.hasAuthorOpenChat ? messageView.authorAvatarTitleText : ''" t-att-aria-label="messageView.hasAuthorOpenChat ? messageView.authorAvatarTitleText : ''"/>
```

Comment:
Difference: `aria-label` was added here and not in previous versions?
Previous version defined alt value correctly instead, why the change?

Reply:
And if the change is justified, probably alt attribute should be at least kept empty instead of removed?

Reply:
Fixed, `alt` is the correct one as it's an img element. All other elements use `aria-label`. I don't know how you noticed that, though :)